cmd/joy/internal_test.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/nestoca/joy/internal/config"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInternal(t *testing.T) {
+	if ok, _ := strconv.ParseBool(os.Getenv("INTERNAL_TESTS")); !ok {
+		t.Skip("Internal tests")
+	}
+
+	configDir, err := os.MkdirTemp("", "")
+	require.NoError(t, err)
+
+	catalogDir := os.Getenv("CATALOG_DIR")
+	require.NotEmpty(t, catalogDir, "catalog repo must be provided")
+
+	ctx := context.Background()
+
+	cfg, err := config.Load(configDir, catalogDir)
+	require.NoError(t, err)
+
+	ctx = config.ToContext(ctx, cfg)
+
+	root := func() *cobra.Command {
+		return NewRootCmd("v9.9.9", PreRunConfigs{})
+	}
+
+	t.Run("setup", func(t *testing.T) {
+		cmd := root()
+		cmd.SetArgs([]string{"setup", "--config-dir=" + configDir, "--catalog-dir=" + catalogDir})
+		require.NoError(t, cmd.ExecuteContext(ctx))
+	})
+
+	t.Run("joy rel validate", func(t *testing.T) {
+		cmd := root()
+		cmd.SetArgs([]string{"release", "validate"})
```

Comment:
Would expect the following, no?
```suggestion
		cmd.SetArgs([]string{"release", "validate"})
		require.NoError(t, cmd.ExecuteContext(ctx))
```